src/main/resources/application.yml

```diff
@@ -29,7 +29,7 @@ springdoc:
     path: /api/swagger
 management:
   server:
-    port: 8081
+    port: 8086
```

Comment:
Why does a pull request for implementing a rate limit change a port number?

Reply:
Hi @MichaelPFrey , thats a bad mistake. I've changed it for local testing purpose. This port-change will be rollbacked in my next commit.